### PR TITLE
Update content blockers informational dialog

### PR DESCRIPTION
**Pull request by Pururun**
- Change programs to apps
- Add custom dns warning
- Add html-formatted text support to info dialog

- - -
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4604)

### CODE REVIEW

File: android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -219,7 +219,10 @@
     contacting certain domains or websites known for distributing ads, malware, trackers and
     more.</string>
     <string name="dns_content_blockers_warning">This might cause issues on certain websites,
-    services, and programs.</string>
+    services, and apps.</string>
+    <string name="dns_content_blockers_custom_dns_warning">
+<![CDATA[Attention: this setting cannot be used in combination with <b>Use custom DNS server.</b>]]>
```
Comment (MrChocolatine) [inline android/app/src/main/res/values/strings.xml:224]: I don't think the full stop should be in bold.

```suggestion
<![CDATA[Attention: this setting cannot be used in combination with <b>Use custom DNS server</b>.]]>
```

File: gui/locales/messages.pot [hunk 2]
```diff
@@ -1627,6 +1627,9 @@ msgstr ""
 msgid "Always-on VPN might be enabled for another app"
 msgstr ""
 
+msgid "Attention: this setting cannot be used in combination with <b>Use custom DNS server.</b>"
```
Comment (MrChocolatine) [inline gui/locales/messages.pot:1630]: Same here.

```suggestion
msgid "Attention: this setting cannot be used in combination with <b>Use custom DNS server</b>."
```

File: android/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Pururun) [reply android/app/src/main/res/values/strings.xml]: Thank you, fixed. :)

**Review by albin-mullvad — CHANGES_REQUESTED** [review]
> Reviewed 22 of 24 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4604)* status: all files reviewed, 3 unresolved discussions (waiting on @MrChocolatine and @Pururun)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/ContentBlockersInfoDialog.kt` line 14 at r2](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4604#-NTmthj41vkuEfg1w7AC:-NTmthj41vkuEfg1w7AD:b-n4sysd) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/4697f4a1e08e98a44a56afc82e01311fc62d307b/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/ContentBlockersInfoDialog.kt#L14)):*
> ```Kotlin
>             buildString {
>                 appendLine(stringResource(id = R.string.dns_content_blockers_info))
>                 append(stringResource(id = R.string.dns_content_blockers_warning))
> ```

Is there a reason for splitting these? I might have forgotten something about it. I guess the important thing is that it's aligned between Android and Desktop to avoid duplicate texts.



_Code quote:_
```Kotlin
                appendLine(stringResource(id = R.string.dns_content_blockers_info))
                append(stringResource(id = R.string.dns_content_blockers_warning))
```

**Review by Pururun — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4604)* status: all files reviewed, 3 unresolved discussions (waiting on @albin-mullvad and @MrChocolatine)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/ContentBlockersInfoDialog.kt` line 14 at r2](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4604#-NTmthj41vkuEfg1w7AC:-NTrTe_n6VEVq5Ew_X2t:b-snmakb) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/4697f4a1e08e98a44a56afc82e01311fc62d307b/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/ContentBlockersInfoDialog.kt#L14)):*
> Is there a reason for splitting these? I might have forgotten something about it. I guess the important thing is that it's aligned between Android and Desktop to avoid duplicate texts.

Talked to Desktop and they want to keep it that way to make it easier to format so I guess we should keep it that way as well.

**Review by albin-mullvad — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4604#-:-NTrVprZ5bc3n9Oect9z:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @MrChocolatine)

**Review by albin-mullvad — APPROVED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4604)* status: all files reviewed, 2 unresolved discussions (waiting on @MrChocolatine)

**Review by albin-mullvad — APPROVED** [review]
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4604)* status: all files reviewed, 2 unresolved discussions (waiting on @MrChocolatine)

**Pull request merged**